uint8 presence_vector
uint8 request_id
uint8 response_code
uint8[80] error_message